README.md

```diff
@@ -0,0 +1,381 @@
+
+![AWS_7.png](/docs/AWS_7.png)
+
+Leave the default selection with t2.micro where the green label says "Free tier eligible" and click on
+"Next: Configure Instance Details" to open the instance details page.
+
+![AWS_7.png](/docs/AWS_7.png)
+
+Just go ahead since we will use the default values on the "Configure Instance Details" page.
+Click on the button "Next: Add Storage".
+
+On the following page you see the storage device settings for your instance.
+Just continue by clicking "Next: Tag Instance", since the default storage will be fine for us.
+
+We dont need to Tag our Instance and we continue by clicking on "Next: Configure Security Group"
+
+In the Configure Security Group page, leave the selected option for "Create a new security group" and
+add rules for protocols and ports by clicking "Add Rule" like in the image below:
+
+![AWS_8.png](/docs/AWS_8.png)
+
+We will add SSH to access and admin the instance (virtual server) by your console.
+The next two rules a pretty straight forward and depends how you want to run openHAB Cloud on your node.
+We recommend to use only HTTPS for Security reason. In this example you also see that we added HTTP.
+
+![AWS_8.png](/docs/AWS_8.png)
+
+After setting up the Security Group, go ahead by clicking the blue "Review and Launch" button.
+
+You will see a summary of your Instance Launch, which will look like this image:
+
+![AWS_9.png](/docs/AWS_9.png)
+
+Just hit the blue "Launch" button and you will be prompted to select or create a key pair
+to connect securely to your instance.
+
+Select the "Create a new key pair" option from the dropdown menue and
+enter a name for your key pair (public and private key) and download your .pem file.
+Warning: Dont loose this file, you will not be able to download it again!
+
+![AWS_10.png](/docs/AWS_10.png)
+
+The key file must not be publicly viewable for SSH to work.
+Open a terminal window and use this command to restrict access rights to the .pem file:
+
+```
+chmod 400 YOUR-PEM-FILENAME.pem
+```
+
+Finally under Instance you can see your instance starting up and running.
+
+
+![AWS_11.png](/docs/AWS_11.png)
+
+There you will find all the needed info to ssh into your node.
+The important info is your Public DNS / Public IP adress.
+
+
+![AWS_13.png](/docs/AWS_13.png)
+
+You can ssh to your instance with the user "ubuntu", see [here](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/managing-users.html) for more info on
+user accounts on linux instances. In terminal window run this command with your instance infos:
+
+```
+ssh -l ubuntu -i YOUR-PEM-FILENAME.pem YOUR-AWS-EC2-PUBLIC-DNS
+```
+
+Now you should be ready with your instance and see the welcome information of ubuntu.
+
+
+
+As next step go to back to the [Quickstart](#quickStart) and follow the installation steps.
+
+
+
+Next we have to configure nginx as webserver and copy the openHAB Cloud nginx config, overriding the default config:
```

Comment:
This part is not specific to EC2 and should be part of the general install notes.  I guess we can also mention that this step is optional as you could just run the node server by itself.